src/api/app/models/kiwi/repository.rb

```diff
@@ -3,7 +3,7 @@ class Kiwi::Repository < ApplicationRecord
   #### Includes and extends
 
   #### Constants
-  REPO_TYPES = ['apt-deb', 'rpm-dir', 'rpm-md', 'yast2'].freeze
+  REPO_TYPES = ['apt-deb', 'rpm-md'].freeze
```

Comment:
Please switch apt-deb with rpm-md, see #3838 (just keep this in mind when you rebase to not introduce the bug again)